tests/observability/metrics/conftest.py

```diff
@@ -663,6 +667,36 @@ def vm_created_pod_total_initial_metric_value(prometheus, namespace):
     )
 
 
+@pytest.fixture()
+def non_evictable_vm_from_template(request, unprivileged_client, namespace):
+    """
+    Create a VM from template with RWO DataVolume created via data_volume_template.
+    This VM will have LiveMigrate eviction strategy with RWO storage, making it non-evictable.
+    The DataVolume is created by the VM itself, which works correctly with WaitForFirstConsumer storage classes.
+    """
```

Comment:
1. please use google-format docstring
2. it has too much info for a fixture  - the name is self-explanatory (i'd rename it to `vm_with_rwo_dv` to make it simple